### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.SignatureVerificationFailedException
  id: SignatureVerificationFailedException
  children:
  - System.IdentityModel.SignatureVerificationFailedException.#ctor
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException
  fullName: System.IdentityModel.SignatureVerificationFailedException
  type: Class
  summary: "Eccezione generata quando si verifica un errore durante l&quot;elaborazione di una firma"
  syntax:
    content: 'public class SignatureVerificationFailedException : System.IdentityModel.Tokens.SecurityTokenException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.IdentityModel.Tokens.SecurityTokenException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> classe."
  remarks: "Questo costruttore inizializza la <xref:System.Exception.Message%2A>proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l&quot;errore.</xref:System.Exception.Message%2A> Questo messaggio prende in considerazione le impostazioni cultura correnti.       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per la nuova istanza.      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore localizzato. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException ();
    parameters: []
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> classe con un messaggio di errore specificato."
  remarks: "Questo costruttore inizializza la <xref:System.Exception.Message%2A>proprietà della nuova eccezione utilizzando il `message` parametro.</xref:System.Exception.Message%2A> Il contenuto di `message` deve essere facilmente comprensibile. Il chiamante del costruttore è necessario per garantire che la stringa sia stata localizzata per la lingua corrente.       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per la nuova istanza.      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio di errore che spiega il motivo dell&quot;eccezione."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> classe con dati serializzati."
  remarks: "Questo costruttore viene chiamato durante la deserializzazione per ricostituire l&quot;oggetto eccezione trasmesso su un flusso."
  syntax:
    content: protected SignatureVerificationFailedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Oggetto <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> oggetto che contiene i dati oggetto serializzati."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Oggetto <xref:System.Runtime.Serialization.StreamingContext>oggetto che contiene le informazioni contestuali sull&quot;origine o destinazione.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> classe con un messaggio di errore specificato e un riferimento all&quot;eccezione interna che è la causa dell&quot;eccezione corrente."
  remarks: "Un&quot;eccezione generata come risultato diretto di un&quot;eccezione precedente deve includere un riferimento all&quot;eccezione precedente nella <xref:System.Exception.InnerException%2A>proprietà.</xref:System.Exception.InnerException%2A> Il <xref:System.Exception.InnerException%2A>proprietà restituisce lo stesso valore passato al costruttore.</xref:System.Exception.InnerException%2A>       Nella tabella seguente vengono illustrati i valori di proprietà iniziali per la nuova istanza.      | Proprietà | Valore |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Il riferimento all&quot;eccezione interna. |   | <xref:System.Exception.Message%2A>| La stringa di messaggio di errore. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio di errore che spiega il motivo dell&quot;eccezione."
    - id: inner
      type: System.Exception
      description: "Il <xref:System.Exception>che è la causa dell&quot;eccezione corrente.</xref:System.Exception> Se il `inner` parametro non è <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, l&quot;eccezione corrente viene generata in un <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blocco che gestisce l&quot;eccezione interna."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.IdentityModel.Tokens.SecurityTokenException
  isExternal: false
  name: System.IdentityModel.Tokens.SecurityTokenException
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException
